compiler/stable_mir/src/mir/body.rs

```diff
@@ -83,6 +83,7 @@ impl Body {
                         Ok(())
                     })
                     .collect::<Vec<_>>();
+                writeln!(w, "{}", pretty_terminator(&block.terminator.kind))?;
```

Comment:
Huh, so we were just not printing the terminator at all? Lol

Reply:
We do print them

Reply:
I'm saying previously, were they not printed?

Reply:
Ah yea, we were not printing them I thought they weren't needed because we don't have successors in `stable_mir` but It looked very sparse, `stable_mir` is still living in stone age